Tools/Create-Azure-Sentinel-Solution/V3/CCP_README.md

```diff
@@ -0,0 +1,275 @@
+# Microsoft Sentinel Solutions Packaging Tool Guidance For CCP Connector
+
+To create a package using V3 packaging tool refer [Link to V3](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.md)
+
+## CCP connector Building Blocks:
+Every CCP connector will have 4 building blocks and should be specified in sequence given below:
+1. Data Connector Definition - Consider this an a UI page shown after deployment to users and contains instruction steps and connectorUiConfig section.
+2. Data Connectors - This will contain details to connect to your REST API and has a poller details.
+3. Data Collection Rule - Specify which data column should be collected
```

Comment:
DCR change it. put meaningful definition of what is DCR and its usage. may be you can point it out official ms doc link for the same

Reply:
updated